### PR TITLE
Chris/ground collision

### PR DESCRIPTION
**Pull request by Chris33871**
Added a working version of gravity for the player
Renamed a couple sprites/functions

### CODE REVIEW

**Comment by Chris33871** [comment]
Don't merge! I'll do it, I anticipate conflicts after merge NJ's brancg

**Comment by 3NJDGZ** [comment]
Let bro cook

File: src/core/player.js
Comment (3NJDGZ) [inline src/core/player.js:1]: applyGravity() {
        this.playerY += 3;
    }

for this, we could have a separate attribute initialised within the player constructor, instead of having just a number. Make it a constant. e.g., 'this.gravity = 3'

File: src/main.js
Comment (3NJDGZ) [inline src/main.js:1]: is it possible to make it so that the player, and bullet positions are updated within the game loop rather than within a separate loop with the ticker? And also when space bar input is checked. Because i feel like the app.ticker.add kind of also acts as another 'game loop' when ultimately there should only be one where everything is updated and checked.

**Review by 3NJDGZ — CHANGES_REQUESTED** [review]
sewey

File: src/main.js
Comment (3NJDGZ) [reply src/main.js:1]: image

File: src/core/player.js
Comment (3NJDGZ) [reply src/core/player.js:1]: yeye no need

**Review by 3NJDGZ — APPROVED** [review]
looks good, everything works

**Pull request merged**
